data.sql

```diff
@@ -5,3 +5,15 @@ INSERT into animal (name, date_of_birth, escape_attempts, neutered, weight_kg)VA
 INSERT into animal (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', '2018-11-15', 2, true, 8);
 
 INSERT into animal (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', '2017-05-12', 5, true, 11);
+
+
+-- Day 2 project
+select * from animals;
```

Comment:
- For proper file organization, please keep everything that is a query that helps to analyze data should be saved in the queries.sql file. Please fix this or remove it if not needed 🙏